sites/www.thunderbird.net/thunderbird/128.0/appeal/index.html

```diff
@@ -0,0 +1,77 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public #
+License, v. 2.0. If a copy of the MPL was not distributed with this # file, You
+can obtain one at http://mozilla.org/MPL/2.0/. -#} {% set active_page =
+"appeal-128" %} {% set page_title_text = _('Experience the future of
+Thunderbird') %} {% extends "includes/base/base.html" %} {% block page_title
+%}{{ page_title_text }}{% endblock %} {% block page_desc %}{{ _('Thunderbird is
+a free email application that’s easy to set up and customize - and it’s loaded
+with great features!') }}{% endblock %} {% block breadcrumbs %}{% endblock %} {%
+block extra_meta %}
```

Comment:
I think you nuked a newline or 5. If you're IDE has jinja template support then assign this file to that.

Reply:
Yeah, I think prettier got a little aggressive :joy:

Reply:
I would exclude prettier from thunderbird-website btw. We don't have a linter setup here.